pkg/controller/kabaneroplatform/landing.go

```diff
@@ -95,8 +95,12 @@ func deployLandingPage(k *kabanerov1alpha1.Kabanero, c client.Client) error {
 		return err
 	}
 
-	// Update the web console's ConfigMap with custom data.
+	// Update the web console's ConfigMap with custom data.  If we could
+	// not find the web console config map, skip it.
 	err = customizeWebConsole(k, c, clientset, config, landingURL)
+	if apierrors.IsNotFound(err) {
```

Comment:
Since this error check is really targeting the call to getWebConsoleConfigMap(c, config) inside customizeWebConsole(), this error check should be put inside customizeWebConsole() and the method should just return cleanly if the console ConfigMap is not found.